include/zephyr/kernel/internal/sched_priq.h

```diff
@@ -10,6 +10,18 @@
 #include <zephyr/sys/dlist.h>
 #include <zephyr/sys/rb.h>
 
+#define K_HIGHEST_THREAD_PRIO (-CONFIG_NUM_COOP_PRIORITIES)
+#define K_LOWEST_THREAD_PRIO CONFIG_NUM_PREEMPT_PRIORITIES
+#define K_IDLE_PRIO K_LOWEST_THREAD_PRIO
+#define K_HIGHEST_APPLICATION_THREAD_PRIO (K_HIGHEST_THREAD_PRIO)
+#define K_LOWEST_APPLICATION_THREAD_PRIO (K_LOWEST_THREAD_PRIO - 1)
+#define K_THRAD_PRIO_NUM (K_LOWEST_THREAD_PRIO - K_HIGHEST_THREAD_PRIO + 1)
+#define PRI_QUEUE_BITMAP_SIZE ((K_THRAD_PRIO_NUM + BIT_NUM - 1) / BIT_NUM)
+#define PRI_BITMAP_ITERATOR PRI_QUEUE_BITMAP_SIZE
+#define BIT_NUM 32
+#define BIT_ARRAY_IDX(prio) ((prio) / BIT_NUM)
+#define BIT_ARRAY_OFFSET(prio) ((prio) % BIT_NUM)
```

Comment:
Likewise these probably want to be in sched.c and not in a header included via the public API